packages/terra-toggle/src/Toggle.jsx

```diff
@@ -39,7 +41,7 @@ class Toggle extends React.Component {
     // within the toggle to disable users from focusing to them
     if (!this.props.isOpen) {
       if (this.contentContainer) {
-        this.contentContainer.classList.add('is-closed');
+        this.contentContainer.setAttribute('data-closed', 'true');
```

Comment:
shouldn't this just be a property managed by `render()` that changes based off state?  If we implement it that way, allt his method does (and the others like it) is toggle the state which causes render to update the data attribute